testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/EnvironmentInspect.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.product.launcher.cli;
+
+import com.github.dockerjava.api.model.Bind;
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Module;
+import io.airlift.log.Logger;
+import io.airlift.units.DataSize;
+import io.trino.tests.product.launcher.Extensions;
+import io.trino.tests.product.launcher.LauncherModule;
+import io.trino.tests.product.launcher.cli.EnvironmentUp.EnvironmentUpOptions;
+import io.trino.tests.product.launcher.env.DockerContainer;
+import io.trino.tests.product.launcher.env.Environment;
+import io.trino.tests.product.launcher.env.EnvironmentConfig;
+import io.trino.tests.product.launcher.env.EnvironmentFactory;
+import io.trino.tests.product.launcher.env.EnvironmentModule;
+import io.trino.tests.product.launcher.env.EnvironmentOptions;
+import io.trino.tests.product.launcher.util.ConsoleTable;
+import org.testcontainers.utility.MountableFile;
+import picocli.CommandLine;
+import picocli.CommandLine.Mixin;
+import picocli.CommandLine.Option;
+
+import javax.inject.Inject;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collection;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.Callable;
+
+import static io.trino.tests.product.launcher.cli.Commands.runCommand;
+import static java.util.Objects.requireNonNull;
+
+@CommandLine.Command(
+        name = "inspect",
```

Comment:
describe would be more appropriate I think

Reply:
inspect feels like "attach to something existing and say how it looks like"

Reply:
Sure. Makes sense :)